package.json

```diff
@@ -18,109 +18,100 @@
+    "gatsby-plugin-sharp": "^4.9.1",
+    "gatsby-plugin-sitemap": "^5.12.1",
+    "gatsby-plugin-styled-components": "^5.22.0",
     "gatsby-plugin-typescript": "^4.20.0",
     "gatsby-redirect-from": "^0.5.0",
-    "gatsby-remark-copy-linked-files": "^5.0.0",
-    "gatsby-remark-images": "^6.0.0",
+    "gatsby-remark-copy-linked-files": "^5.12.1",
+    "gatsby-remark-images": "^6.12.1",
     "gatsby-remark-relative-images": "^2.0.2",
-    "gatsby-source-filesystem": "^4.0.0",
-    "gatsby-theme-material-ui": "^5.3.0",
-    "gatsby-transformer-json": "^4.0.0",
-    "gatsby-transformer-remark": "^5.0.0",
-    "gatsby-transformer-sharp": "^4.0.0",
-    "graphql": "^15.0.0",
-    "graphql-tag": "^2.10.3",
+    "gatsby-source-filesystem": "^4.9.1",
+    "gatsby-transformer-sharp": "^4.9.0",
+    "graphql-tag": "^2.12.6",
     "is-absolute-url": "^2.1.0",
-    "js-cookie": "^2.2.1",
+    "js-cookie": "^3.0.1",
     "lodash": "^4.17.21",
     "lodash.groupby": "^4.6.0",
     "lodash.startcase": "^4.4.0",
     "path-browserify": "^1.0.1",
     "prism-react-renderer": "^1.3.5",
+    "process": "^0.11.10",
     "prop-types": "^15.8.1",
-    "react": "^17.0.2",
+    "react": "^17.0.1",
     "react-body-classname": "^1.3.1",
     "react-dom": "^17.0.2",
     "react-emotion": "^10.0.0",
-    "react-helmet": "^6.1.0",
+    "react-helmet": "^5.2.1",
     "react-live": "^2.2.3",
     "react-markdown": "^8.0.3",
     "react-scrollspy": "^3.4.3",
     "remark-containers": "^1.2.0",
     "sass": "^1.54.4",
-    "slugify": "^1.6.5",
+    "slugify": "^1.3.6",
     "styled-components": "^5.3.0",
-    "typescript": "^4.7.4",
     "unist-util-visit": "^2.0.3"
   },
   "devDependencies": {
-    "@types/node": "^18.7.2",
-    "@types/react": "^18.0.17",
-    "@types/react-dom": "^18.0.6",
-    "@types/react-helmet": "^5.0.15",
-    "@typescript-eslint/eslint-plugin": "^5.33.0",
-    "@typescript-eslint/parser": "^5.33.0",
-    "eslint": "^8.21.0",
+    "@types/js-cookie": "^3.0.1",
+    "@types/mdx-js__react": "^1.5.5",
+    "@types/node": "^17.0.21",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.13",
+    "@types/react-helmet": "^6.1.5",
+    "@typescript-eslint/eslint-plugin": "^5.20.0",
+    "@typescript-eslint/parser": "^5.20.0",
+    "eslint": "^7.16.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-react": "^7.30.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "prettier": "^2.7.1"
+    "eslint-config-standard": "^14.1.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.29.4",
+    "eslint-plugin-react-hooks": "^4.4.0",
+    "prettier": "^1.17.1",
+    "typescript": "^4.6.2"
```

Comment:
It seems most of these are package _downgrades_ - including a point release of TypeScript and a patch release of React. Even a major version of Prettier. I understand the rest of the package downgrades, but why these three?

Reply:
@NickLanam @htroisi Upgraded some package back to latest version as were in first commit (https://github.com/pixie-io/docs.px.dev/pull/218/commits/6a40f7f26fa33de31ce5a347deba16a3afe8dd11#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)